links + updated reports page and more (#495)
OctoMind-dev/mintlify-docs: manage-tests/test-reports.mdx

<Frame caption="Example of Octomind test results in a commit comment, 09/2023">
  <img
    src="/images/Octomind-test-results-commit-comments-GitHub-example.png"
    alt="Example of Octomind test results in a github pull request comment"
  />
</Frame>

There will be a row for each test with the following information:

| column      | description                                               |
| ----------- | --------------------------------------------------------- |
| description | name of the test case                                     |
| status ✅   | execution successful                                      |
| status ⌛️   | execution in progress                                     |
| status ❌   | execution failed                                          |
| status 💔   | something went wrong on our side during execution         |
| details     | link which takes you to the test case details (see below) |

## Test report details

When you click on a specific test report in the app or on the link in your commit comment, a `test report overview` with all the results and more details will open.

> If trigger manually, you will find the user who triggered it and when. When triggered via CI integration, you will find a link to the `commit` that triggered it.

Reviewer: ```suggestion
If triggered manually, you will find the user who triggered it and when. When triggered via CI integration, you will find a link to the `commit` that triggered it.
```

<Frame caption="Example of a Octomind test report, 05/2025">
  <img
    src="/images/manage/test-report.png"
    alt="Example of a Octomind test report"
  />
</Frame>

## Test result details

If you click on one of the tests within the test report, you will find details about the execution and its results.

> It will give showcase test step timeline to understand what the execution did exactly. You can swipe through the steps back and forth to see all the interactions Octomind executed for this test run.

Reviewer: ```suggestion
It will show the test step timeline to understand what the execution did exactly. You can swipe through the steps back and forth to see all the interactions Octomind executed for this test run.
```

<Frame caption="timeline of a broken test in test result detail showing what happened during the test run, 5/2025">
  <img
    src="/images/manage/timeline-test-result.gif"
    alt="snapshots carousel in test reports"
  />
</Frame>

The test result detail gives you insight into pre-classified root cause of a test failure, agent commentary, original prompt, you can compare test runs, check error log and more.

> It gives you all the necessary info to inspect broken test a fix the app or your test, depending on what caused it. [This is how you do it.](/maintain-tests/inspect-broken-tests)

Reviewer: ```suggestion
It gives you all the necessary info to inspect broken test and fix the app or your test, depending on what caused it. [This is how you do it.](/maintain-tests/inspect-broken-tests)
```

<Frame caption="One view with all information to inspect what went wrong with a broken test, 4/2025">
  <img
    src="/images/maintain/timeline-inspect-options.png"
    alt="One view with all information to inspect what went wrong with a broken test"
  />
</Frame>

## Debugging a failed test

> In the `maintain tests` section of this documentation, we explain [why tests fail](/maintain-tests/why-tests-fail), how do you [inspect the cause](/maintain-tests/inspect-broken-tests) and how you can [fix broken tests](/maintain-tests/auto-fix).

Reviewer: ```suggestion
In the `maintain tests` section of this documentation, we explain [why tests fail](/maintain-tests/why-tests-fail), how you [inspect the cause](/maintain-tests/inspect-broken-tests) and how you can [fix broken tests](/maintain-tests/auto-fix).
```
